scripts/setup-tests.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function () {
+  process.env.NODE_ENV = 'test'
+}
```

Comment:
Why is it required?

Isn't it the default env for Jest?

Reply:
Hmm. It's probably because I had `export NODE_ENV=development` in my `.zprofile`. Should I remove it?

Reply:
Yes please :slightly_smiling_face: